README.md

```diff
@@ -40,6 +40,17 @@ cp boxes.yaml.example boxes.yaml
 vagrant up centos7-devel
 ```
 
+### Poor man's DNS a.k.a /etc/hosts
+
+For the mulch-host setup, one of the easiest way of making the name
```

Comment:
No, I really meant `mulch`:) Just kidding. Updated + added info about the default domain the hosts are available in.